Certificate management (#2)
Devolutions/picky-ps: README.adoc

PS > Request-Certificate -Subject MySubject -PickyApiKey 123456
----

The ID of the certificate will be returned as a result.

[source, sh]
----
6462eb3e-addb-499a-9727-894994b22ed8
----

=== Get Certificates

> The list of signed certificates can be show with the command `Get-LocalCertificates`

Reviewer: ```suggestion
The list of signed certificates can be shown with the command `Get-LocalCertificates`
```

[source, sh]
----
PS > Get-LocalCertificates
----

The result:

[source, sh]
----
Issuer                Subject     ID
------                -------     --
CN=Wayk.net Authority John        2c082069-5281-4ed4-aadb-cd0fbb15bc30
CN=Wayk.net Authority Marie       39f69f39-6eec-4aa3-9663-4a4ae37b90b5
----

=== Save Certificate on Server

> You can save certificate who have been signed on the backend of the Picky server with the command `Save-CertificateOnServer`

Reviewer: ```suggestion
You can save certificates that have been signed on the backend of the Picky server with the `Save-CertificateOnServer` command.
```

> The parameters are needed:

Reviewer: ```suggestion
Required parameters are:
```

- `PickyApiKey`, the Api Key of your Picky Server.

- `CertificateID`, The ID of the certificate

> - `PickyUrl`, (Facultative) the url with the port for your Picky Server, the default value is 'http://127.0.0.1:12345'

Reviewer: ```suggestion
- `PickyUrl`, (Facultative) url and port to your Picky Server, the default value is 'http://127.0.0.1:12345'
```
